libevm/VMCalls.cpp

```diff
@@ -120,6 +120,9 @@ void VM::caseCreate()
 	uint64_t initOff = (uint64_t)m_SP[1];
 	uint64_t initSize = (uint64_t)m_SP[2];
 
+	// Clear the return data buffer. This will not free the memory.
+	m_returnData.clear();
+
```

Comment:
When the contract initialization executes `REVERT`, `m_returnData` has to be set.